src/lib/action/keyboard.ts

```diff
@@ -9,39 +9,54 @@ export type keyMap = { [key: string]: Function }
  * Invalid modifer keys will be ignored.
  */
 const sanitise = (keyString: string): string => {
-    let keys = keyString.split('+').map(k=>(capitalise(k)))
-    let key = keys.pop()
-    let alt = keys.includes("Alt") ? 'Alt+' : ''
-    let ctrl = keys.includes("Control") || keys.includes("Ctrl") ? 'Control+': ''
-    let meta = keys.includes("Meta") || keys.includes("Super") || keys.includes("Command") || keys.includes("Cmd") || keys.includes("Win") ? 'Meta+' : ''
-    let shift = keys.includes("Shift") ? 'Shift+': ''
-    return alt+ctrl+meta+shift+key
+	const keys = keyString.split('+').map((k) => capitalise(k));
+	const key = keys.pop();
+	const alt = keys.includes('Alt') ? 'Alt+' : '';
+	const ctrl = keys.includes('Control') || keys.includes('Ctrl') ? 'Control+' : '';
+	const meta =
+		keys.includes('Meta') ||
+		keys.includes('Super') ||
+		keys.includes('Command') ||
+		keys.includes('Cmd') ||
+		keys.includes('Win')
+			? 'Meta+'
+			: '';
+	const shift = keys.includes('Shift') ? 'Shift+' : '';
+	return alt + ctrl + meta + shift + key;
+};
+
+function getSanitisedShortcutsFromKeyMap(keyMap: keyMap) {
```

Comment:
Oh and add a small descriptive comment
```
/**
 * Takes a keyMap and returns it with its key strings sanitised.
 */
 ```

Reply:
> I think we can call this func `sanitiseKeyMap`. I'm not fond of long function names, especially when fns that are not exported so are always used in context. Fn takes in a keyMap and returns the same keyMap but sanitised, so the short name fits.
> 
> As you might've noticed from the codebase I have a preference for writing funcs as arrow functions as long as they don't need `this` (or are class methods/constructors). If you don't mind we can do that everywhere. If you have a good reason why not, I'm open to change, but it should be consistent.

I agree on the renaming! I have a personal preference for defining functions with the function keyword. I feel that a lot of people adopted the arrow syntax just because it was new - not because it gave any benefit. I only use arrow functions for the cases where they are needed.

To me its much cleaner and easier to instantly see that this is a function
```ts
function sanitiseKeyMap(keyMap: keyMap) {
	// stuff
}
```

... compared to this.
```ts
const sanitiseKeyMap = (keyMap: keyMap) => {
	// stuff
}
```

Reply:
Yeah I have not been convinced that there is a good reason to choose one of the other than looks (you can have the global execution context as rule or exception, in most cases it doesn't matter).
I like the clean look of the arrow fns, but there is definitely something to be said for  the expressiveness in your case.
To be honest I do not care too much either way, the only thing I care about is consistency over the codebase. Feel free to keep it like this and I'll align the others sometime soon.